File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/AppointmentResponse.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.Resource;

public class AppointmentResponse : DomainResource, IResource
{
    public string? End { get; set; }
    public Base.ResourceReference? Actor { get; set; }
    public string? Start { get; set; }
    public string? Comment { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public required Base.ResourceReference Appointment { get; set; }
    public Base.CodeableConcept[]? ParticipantType { get; set; }
    public required string ParticipantStatus { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/AppointmentResponse.py
from typing import Optional, List
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier

class AppointmentResponse(DomainResource):
    end: Optional[str] = None
    actor: Optional[Reference] = None
    start: Optional[str] = None
    comment: Optional[str] = None
    identifier: Optional[List[Identifier]] = None
    appointment: Reference
    participant_type: Optional[List[CodeableConcept]] = None
    participant_status: str